masonite/providers/RouteProvider.py

```diff
@@ -93,7 +93,13 @@ def boot(self, WebRoutes, Route, Request, Environ):
                 # Get the data from the route. This data is typically the output
                 #     of the controller method
                 if not request.redirect_url:
```

Comment:
Ok so sometimes I want the view to return a string so it won't always have a `.rendered_template` attribute. We can do this simply by importing `from masonite.view import View` at the top and right here you'll need to do:

```python
# Get the data from the route. This data is typically the output
#     of the controller method
if not request.redirect_url:
    response = self.app.resolve(route.output)

    if isinstance(response, View):
        response = response.rendered_template


    self.app.bind(
        'Response',
        router.get(
            route.route,
            response
         )
    )
```

this will allow us to do something like:

```python
def show(self):
    return 'this'
```

which we were able to do before but now we are not since we changed this route provider.